backend/app/services/shared_task.py

```
@@ -895,6 +895,22 @@ def get_public_shared_task(
+        )
+
+        result_wrapper = {
+            "subtasks": [
+                {"result": public_subtask.result} for public_subtask in public_subtasks
+            ]
+        }
+        refresh_extended_video_result_urls(result_wrapper, user_id)
+        for public_subtask, refreshed in zip(
+            public_subtasks,
+            result_wrapper["subtasks"],
+        ):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)